data/yam/agama/auto/alp_leap16.sh

```diff
@@ -0,0 +1,8 @@
+set -ex
+
+/usr/bin/agama config set software.product=Leap16
+/usr/bin/agama config set user.userName=joe user.password=doe
+/usr/bin/agama config set root.password=nots3cr3t
+/usr/bin/sleep 30
+/usr/bin/agama install
+/sbin/reboot
```

Comment:
we removed this step in previous PR.
Also we should use as user bernhard and password `nots3cr3t`. Please check recent work.